db/migrate/20230430025441_add_archive_to_families.rb

```diff
@@ -1,7 +1,6 @@
 class AddArchiveToFamilies < ActiveRecord::Migration[7.0]
   def up
-    add_column :families, :archived, :boolean
-    change_column_default :families, :archived, false
+    add_column :families, :archived, :boolean, nill: false, default: false
```

Comment:
This is a typo, no? `nill`

Reply:
Good catch!

Reply:
Fixed it and updated the branch. Should be set now!